tests/test_utils.py

```diff
@@ -28,6 +28,20 @@ def test_does_not_add_path_if_it_does_not_exist(self):
         assert_that(self.not_existing_path in sys.path).is_false()
 
 
+class TestLoadModule(unittest.TestCase):
+    def setUp(self):
+        self.test_module_name = "os"
```

Comment:
This kind of gives away the problem. You are expecting a module name where module_name is really a `.py` file name.

Reply:
Do you have any idea about a better test_module_name?